lib/sentry/client.ex

```diff
@@ -157,6 +163,22 @@ defmodule Sentry.Client do
     {endpoint, public_key, secret_key}
   end
 
+  def maybe_call_after_send_event({:ok, _} = result, event) do
+    case Application.get_env(:sentry, :after_send_event) do
```

Comment:
they each call the configured function differently because of different arities, so this makes less sense now right?